### DISCUSSION TITLE
[WS2021] Homework 7 Discussion

### DISCUSSION THREAD

**Discussion by changkun** [mimuc/gp-ws2021#7] [Discussion] [answered]
This thread is open for the discussion of homework 7.

**Comment by nico778** [comment]
Not a homework question, but a question in regard to the paper [Wang et al. 2019] "Dynamic graph cnn for learning on point clouds." 
Are they using decision trees(kd tree) or are they using a simple knn algorithm and achieve the point grouping only via the sequential layers?

**Reply by changkun** [reply]
According to their source code, it is a simple kNN to extract graph features:

https://github.com/WangYueFt/dgcnn/blob/20fdb459ca5d10fe8aba1d296e66340f65990b85/pytorch/model.py#L21-L27

**Reply by nico778** [reply]
thanks for the clarification, I will have a look

**Comment by changkun** [comment, marked as answer]
Close due to the end of the submission period.
